### ISSUE TITLE
Cannot install with composer because of guzzle 6.x

### ISSUE THREAD

**Issue by iflorespaz** [RusticiSoftware/scormcloud-api-v2-client-php#17]
Hello,

I cannot install and provide your solution to my customers because the library has required guzzle 6, as you know guzzle 7 has been release the past year : https://laravel-news.com/guzzle-7-released.

I think the PHP library should be updated in order to work in future PHP releases 7.x and 8.x because 5.x is not supported anymore.

```
  Problem 1
    - rustici-software/scormcloud-api-v2-client-php[1.1.0, ..., 1.1.1] require guzzlehttp/guzzle ~6.0 -> satisfiable by guzzlehttp/guzzle[6.0.0, ..., 6.5.5].
    - You can only install one version of a package, so only one of these can be installed: guzzlehttp/guzzle[6.0.0, ..., 6.5.5, 7.0.0, ..., 7.3.0].


```

Could you tell me if you have plans to upgrade you library ?

Best regards,
Isaac Flores

**Comment by bmewburn** [comment]
Agree. 2.0 version with guzzle 7 would be great. Here's a PR https://github.com/RusticiSoftware/scormcloud-api-v2-client-php/pull/18 . Thanks for your work.

**Comment by iflorespaz** [comment]
Waw good work, Thanks.

**Comment by mjschuetze102** [comment]
Apologies for the delay here. We are currently in the process of updating all of our client libraries. As part of this process, we went through and made sure that our software would continue to work with the most up-to-date dependencies/ languages (for this one in particular GuzzleHttp 7.3). Unfortunately, [swagger-codegen](https://github.com/swagger-api/swagger-codegen) (what we use to generate our client libraries) has not yet updated their templates to support GuzzleHttp 7. As such, we are extensively testing the update to make sure that we have modified all the necessary codepaths.
